example/run_airl.py

```diff
@@ -205,9 +203,10 @@
 
 if args.pretrain:
     with measure('bc pretrain'):
-        _ = behavior_clone.train(
-            expert_traj, pol, optim_pol, args.bc_batch_size, args.bc_epoch
-        )
+        for _ in range(args.bc_epoch):
+            _ = behavior_clone.train(
+                expert_traj, pol, optim_pol, args.bc_batch_size
+            )
```

Comment:
This was bug, behavior_clone.train doesn't have the argument of epoch.
Therefore, it is a bug that args.bc_epoch is passed to behavior_clone.train.
Then, I have fixed.